ecommerce/extensions/checkout/views.py

```diff
@@ -194,6 +194,9 @@ def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
             'has_enrollment_code_product': has_enrollment_code_product,
             'disable_back_button': self.request.GET.get('disable_back_button', 0),
         })
+        context.update({
+            'user_full_name': self.request.user.get_full_name()
+        })
```

Comment:
@alfredchavez, can you explain why this is needed?

Reply:
@lgp171188 I've added it so full name can be used on the receipt page. As I tried with user.full_name and order.user.full_name and I could not get to display the value